### PR TITLE
adding day 2 transition sentences

### PR DESCRIPTION
**Pull request by ChristinaLK**
Last of the "smoothing" PRs!

### CODE REVIEW

File: _episodes/13-second-welcome.md [hunk 1]
```diff
@@ -17,20 +17,33 @@ keypoints:
...
+- understand the importance of a respectful and inclusive classroom environment; commit to
+creating such an environment; and be able to
+identify and implement Carpentries policies and general practices to accomplish this.
+- practice and develop skills in the teaching methods used in Carpentries workshops
+- learn enough about the Carpentries organization to know where to go for help,
+how to start organizing a workshop, and how to get involved with community activities.
 
+Yesterday we focused on general aspects of educational psychology and pedagogy and
+how to create a positive classroom as environment, addressing the first two goals.
+We started to address the third goal by discussing the important of lesson study
```
Comment (karenword) [inline _episodes/13-second-welcome.md]: "importance"

File: _episodes/13-second-welcome.md [hunk 2]
```diff
@@ -17,20 +17,33 @@ keypoints:
...
+and observation, and by providing an opportunity to teach and
+receive feedback. We will continue that skill development process today, as we
+focus on the specifics of teaching Software Carpentry, Data Carpentry, and Library Carpentry workshops.
+We will review specific
+teaching practices we follow at our workshops and practice using some of them. One of the most important practices is participatory live coding. You rarely
+find this approach in a current university setting so we'll spend some time practicing this skill.
```
Comment (karenword) [inline _episodes/13-second-welcome.md]: There are a few takes on why we practice -- I see these noted elsewhere but here might be good to prepare them to think in all the ways. One of them is the rehearsal of live coding itself. Others are that we are rehearsing the process of reflective practice, as well as developing skills in providing constructive feedback.

File: _episodes/17-performance.md [hunk 3]
```diff
@@ -10,6 +10,11 @@ objectives:
 keypoints:
 - "(Reflective) Practice makes perfect."
 ---
+
+Continuing our theme of developing practical Carpentries teaching skills,
+this section provides another chance to practice live coding, observe others
+and give/receive feedback.
+
```
Comment (karenword) [inline _episodes/17-performance.md]: maybe add something about incorporating or responding to feedback?

File: _episodes/13-second-welcome.md (hunk 2 above)
Comment (ChristinaLK) [reply _episodes/13-second-welcome.md]: good point. I'll add those aspects as well.

File: _episodes/17-performance.md (hunk 3 above)
Comment (ChristinaLK) [reply _episodes/17-performance.md]: ah, yes!

**Comment by ChristinaLK** [comment]
Done! do another quick check?

**Pull request merged**